ProjectCoimbra.UWP/Project.Coimbra.Midi/Models/InstrumentInfo.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed under the MIT License.
+
+namespace Coimbra.Midi.Models
+{
+	using System;
```

Comment:
This file uses a mix of space and tab indentation. Please use spaces for consistency with the rest of this repo (and StyleCop standards, which is what we're using)

Reply:
Sorry VS messed that up. Fixing for all files now.